api/server/router/container/backend.go

```diff
@@ -15,7 +13,7 @@ import (
 // execBackend includes functions to implement to provide exec functionality.
 type execBackend interface {
 	ContainerExecCreate(config *types.ExecConfig) (string, error)
-	ContainerExecInspect(id string) (*exec.Config, error)
+	ContainerExecInspect(id string) (interface{}, error)
```

Comment:
I'm still learning go, but seeing `interface{}` like this seems to be kind of a yellow flag to me. Here's how I'm interpreting it:

It's a return value here, so as long as it can be serialized to json, it should be okay. Is there some 'serializable' tag we can use, or is jsonification of objects sufficiently magic that we do not need to worry about it?